chpl/chpl-service/src/main/java/gov/healthit/chpl/form/AllowedResponse.java

```diff
@@ -0,0 +1,32 @@
+package gov.healthit.chpl.form;
+
+import org.apache.commons.collections4.Equator;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class AllowedResponse {
+    private Long id;
+    private String response;
+    private Integer sortOrder;
+
+    public static class AllowedResponseByIdEquator implements Equator<AllowedResponse> {
+
+        @Override
+        public boolean equate(AllowedResponse o1, AllowedResponse o2) {
```

Comment:
Is this supposed to be `equals`? If so, and the functionality works without override `equals`, do we need this at all?

Reply:
It is from Apache Collection Commons - it is used in conjunction with `CollectionUtils.isEqualCollection( ... )` to determine if a collection of objects is equal, w/o respect to order.